posthog/warehouse/api/external_data_source.py

```diff
@@ -461,9 +492,9 @@ def destroy(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         ).all()
         for job in all_jobs:
             try:
-                delete_data_import_folder(job.folder_path)
+                delete_data_import_folder(job.folder_path())
             except Exception as e:
-                logger.exception(f"Could not clean up data import folder: {job.folder_path}", exc_info=e)
+                logger.exception(f"Could not clean up data import folder: {job.folder_path()}", exc_info=e)
```

Comment:
It needed to be wrapped in a `sync_to_async` call in a test because calling the property was failing from async issues - couldn't figure out how to do the wrapping as a property, and so just changed it to a func instead